arcaflow_plugin_stressng/stressng_schema.py

```diff
@@ -2,12 +2,15 @@
 
 import typing
 import enum
+import re
 from dataclasses import dataclass
 
-from arcaflow_plugin_sdk import plugin, schema
+from arcaflow_plugin_sdk import plugin, schema, validation
 from arcaflow_plugin_sdk import annotations
 
 
+bytes_pattern = re.compile(r"^[1-9]\d*\.?\d*[KkMmGgTt]$")
```

Comment:
The stressng parser is weird. I'd really expect "0.1G" and ".1G" to work, but apparently even the latter doesn't? Huh.